src/applications/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -52,15 +52,15 @@ function UserInfoSection({ enrollmentData = {}, showCurrentAsOfAlert }) {
   }
 
   let entitlementInfo;
-  const originalEntitlement = enrollmentData.originalEntitlement;
-  const usedEntitlement = enrollmentData.usedEntitlement;
-  const remainingEntitlement = enrollmentData.remainingEntitlement;
+  const { originalEntitlement } = enrollmentData;
+  const { usedEntitlement } = enrollmentData;
+  const { remainingEntitlement } = enrollmentData;
```

Comment:
I believe this can be simplified to 

`const { originalEntitlement, usedEntitlement, remainingEntitlement }  = enrollmentData;`